codersdk/deployment.go

```diff
@@ -1353,6 +1353,16 @@ Write out the current server configuration to the path specified by --config.`,
 			Value:       &c.GitAuthProviders,
 			Hidden:      true,
 		},
+		{
+			Name:        "Custom Tunnel Host",
+			Description: `Hostname of HTTPS server that runs https://github.com/coder/wgtunnel. By default, this will pick the best available wgtunnel server hosted by Coder. e.g. "tunnel.example.com".`,
+			Flag:        "tunnel-host",
```

Comment:
The fact that the tunnel is wireguard is transparent to the user, so adding `wg` seems to me like it would confuse people. This is a hidden var since we don't intend on many people using it though so maybe `wgtunnel-host` is fine unless we want to unhide it.

Reply:
I like `wgtunnel-host`.